src/plugins/ctd/mapping.py

```diff
@@ -6,12 +6,30 @@ def get_customized_mapping(cls):
         "taxid": {
             "type": "integer"
         },
+        "source": {
+            "normalizer": "keyword_lowercase_normalizer",
+            "type": "keyword"
+        },
+        "name": {
+            "normalizer": "keyword_lowercase_normalizer",
+            "type": "keyword"
+        },
+        "description": {
+            "type": "text"
+        },
```

Comment:
let's add 
`"copy_to": [
                        "all"
]` to `name` and `description` fields.

Reply:
also to `chemical_name`, `mesh`, and `cas`